html_image_url_extractor/tests/test_extractor.py

```diff
@@ -1,18 +1,26 @@
 # -*- coding: utf-8 -*-
-# © 2016 Grupo ESOC Ingeniería de Servicios, S.L.U. - Jairo Llopis
+# Copyright 2016-2017 Jairo Llopis <jairo.llopis@tecnativa.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
+import logging
+
 from lxml import etree
 from odoo.tests.common import TransactionCase
+from ..models import ir_fields_converter
 
 
 class ExtractorCase(TransactionCase):
     def setUp(self):
         super(ExtractorCase, self).setUp()
-
+        # Silent errors to keep beautiful logs and make CI pass
+        ir_fields_converter._logger.disable(logging.ERROR)
```

Comment:
And you could avoid this extra code since that odoo supports add filtered logger using [mute_logger](https://github.com/odoo/odoo/blob/15da3356a088609530b1943e6f24d734c00ccced/odoo/tools/misc.py#L891-L924)